[*] running method core_negotiate_tlv_encryption
[*] Negotiating TLV encryption
[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100bec7ee5c11a0f6b9ebfd8b7690e6362106f177b3939e5bdc736f617843710d063cd1d22aa4f0ee43fc13230d3bf62dc58491b07982362abd95739143eb8a3fbb0e2f1618ea61101c7103cd8a3e0b5a7fd2b9576b257dfa7307b8d7344901ff81dc35cd0951eb560b3efdc86c1685c939b43c85ea3dc191fac313759a6e01064163f5ecca16bf848cf000539b60e8844b9caf70b9f8145cb6e1e7ea1b7153ec8047ad924848a6359b2eedfbb9acb8ebcdf5a57299525708dd94942ddd351912b7e9aced23437701c0d6bad97bd8a913b73401a3b172fc68006fbcb8e7818d861f2517e21652dbc1a65a6ba6e983e6e724b4027d6d1474101ae21d957f46642ee90203010001
[*] AES key: 0xef4e56b41a3c3eb5f455633e5924021f4f60b93746a38615a7489a6810f3fdef
[*] Encrypted AES key: 0x758d4bed5388e5aa6185d08647fcf991162a62e72f12ee1101c8a3a8f72eed6029c1ffd8fece6ed07b41ce99c02484b9f00719c48bea0ad434d7269d921e1c65f20a2665f94a6ca9853d4bd1b26ab98ae8a4ce63f40e22491f84ff510c1a44ecd9c3eb3f2d4bd63e3f1a9e1719543f4775052ad31d86f5992d0fa720e0193686fc46ca6d751fdf0ae50011d14922656438233fcc2e50100086a9a471bdbf79a7ab566085084816f104bf4e3007a97ef607ae8989b653730b2fe1ee51283aeffdd8cba0bd561621fa47bd98ca335e7a7aa16e5b2516af7af5b8b9f76073a17b5dc259d258f37cbcecefd9b837005c10c91092d320f18a06a65a8039e0d2356c9b
[*] TLV encryption sorted
[*] sending response packet
[*] running method core_set_session_guid
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_loadlib
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getuid
[*] sending response packet
[*] running method stdapi_sys_config_sysinfo
[*] sending response packet
[*] running method core_set_uuid
[*] sending response packet
[*] running method stdapi_net_config_get_interfaces
[*] sending response packet
[*] running method stdapi_net_config_get_routes
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_separator
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-pzgjiqun'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 1 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 2 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 3 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 3
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 4 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 4
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 5 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 5
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 6 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 6
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 7 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 7
[*] sending response packet
[*] running method stdapi_fs_ls
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo KFZZEOFN']
[*] added process id: 2084, handle: 1
[*] added channel id: 8 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 9 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 9
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
[*] added process id: 2085, handle: 2
[*] added channel id: 10 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 11 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 11
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
[*] added process id: 2092, handle: 3
[*] added channel id: 12 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 3
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 13 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 13
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 14 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 14
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 15 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 15
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 16 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 16
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 17 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 17
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 18 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 18
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 19 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 19
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method stdapi_fs_file_move
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
